radix-engine/tests/transaction_commit_rollback.rs

```diff
@@ -30,10 +30,10 @@ fn test_state_track_success() {
     // Assert
     receipt.expect_commit_success();
     assert_eq!(
-        22,
+        25,
```

Comment:
This feels like a fragile (non-resistant to refactoring) test. Can we just remove this assertion or replace with something more resistant to refactoring?

Reply:
Definitely.